src/pytest_plugins/forks/forks.py

```diff
@@ -153,6 +170,20 @@ def add_values(self, metafunc: Metafunc, fork_parametrizer: ForkParametrizer) ->
         fork_attribute_name="precompiles",
         parameter_name="precompile",
     ),
+    CovariantDescriptor(
+        marker_name="with_all_evm_code_types",
+        description="marks a test to be parametrized for all EVM code types at parameter named"
+        " evm_code_type of type EVMCodeType",
+        fork_attribute_name="evm_code_types",
+        parameter_name="evm_code_type",
+    ),
+    CovariantDescriptor(
+        marker_name="with_all_call_opcodes",
+        description="marks a test to be parametrized for all *CALL opcodes at parameter named"
+        " call_opcode, and also the appropriate EVM code type at parameter named evm_code_type",
+        fork_attribute_name="call_opcodes",
+        parameter_name="call_opcode,evm_code_type",
```

Comment:
Also if I already have parametrisation with this name how the collision resolve?

Reply:
You have to manually remove it before adding this marker to a test.

You can see an example of this with the 4844 pre-compile test, where it previously had this:
https://github.com/ethereum/execution-spec-tests/blob/28f376b5264133c1eefedbc5d8006ae557924a15/tests/cancun/eip4844_blobs/test_point_evaluation_precompile.py#L450-L458
And was replaced with:
https://github.com/ethereum/execution-spec-tests/blob/a87bb09025d8472c9efacb67ccd839f003211736/tests/cancun/eip4844_blobs/test_point_evaluation_precompile.py#L497

Also all instances of `call_type` had to be renamed to what the marker actually parametrizes: `call_opcode`.